### PR TITLE
Pepper 999 and Pepper 1000 add automation for saliva and blood kit for research studies

### PR DESCRIPTION
**Pull request by Rinchan20**
Added Saliva and Blood kit upload for pancan - will add it for prostate/MPC once Pepper-1300 is done (it has methods that would make it simpler)

### CODE REVIEW

**Comment by Rinchan20** [comment]
forgot to add to blood test - adding

File: playwright-e2e/tests/dsm/kitUploadFlow/blood-kit-upload-flow.spec.ts [hunk 1]
```diff
@@ -51,6 +57,15 @@ test.describe.serial('Blood Kits upload flow', () => {
   for (const study of studies) {
     test(`Should upload a single kit for one participant @functional @dsm @${study}`, async ({page}, testInfo) => {
       testResultDir = testInfo.outputDir;
+      if (clinicalStudies.includes(study)) {
```
Comment (zyme) [inline playwright-e2e/tests/dsm/kitUploadFlow/blood-kit-upload-flow.spec.ts:60]: Good spot for ternary operator

Comment (zyme) [reply playwright-e2e/tests/dsm/kitUploadFlow/blood-kit-upload-flow.spec.ts:60]: `isClinicalStudy = clinicalStudies.includes(study) ? true : false;`

**Review by zyme — APPROVED** [review]

File: playwright-e2e/tests/dsm/kitUploadFlow/blood-kit-upload-flow.spec.ts (hunk 1 above)
Comment (Rinchan20) [reply playwright-e2e/tests/dsm/kitUploadFlow/blood-kit-upload-flow.spec.ts:60]: I tried that before but it seems to make lint complain:
Screenshot 2024-06-03 at 4 24 04 PM

**Pull request merged**